### PR TITLE
node: update sequelize and knex docs for TraceProvider

### PR DESCRIPTION
**Pull request by aabmass**
These have a bit more than #70 #69 which can be closed after this PR.

- update sequelize documentation with TraceProvider and samples
- update knex documentation with TraceProvider and samples

### CODE REVIEW

File: content/node/knex/_index.md [hunk 1]
```diff
@@ -93,37 +93,51 @@ In the database server logs, the comment's fields are:
...
-db_driver|<div style="text-align: center">&#10060;</div>
+##### `include`
+A map of values to be optionally included in the SQL comments.
+
+| Field           | On by default |
+| --------------- | ------------- |
+| db_driver       | No            |
+| route           | Yes           |
+| traceparent     | No            |
+| tracestate      | No            |
```
Comment (odeke-em) [inline content/node/knex/_index.md:120]: Please revert the checkmark and x as they are more visually indicative (I found) than Yes or No.

File: content/node/sequelize/_index.md [hunk 2]
```diff
@@ -93,44 +92,60 @@ In the database server logs, the comment's fields are:
...
+##### `include`
+A map of values to be optionally included in the SQL comments.
+
+| Field           | On by default |
+| --------------- | ------------- |
+| client_timezone | No            |
+| db_driver       | No            |
+| route           | Yes           |
+| traceparent     | No            |
+| tracestate      | No            |
```
Comment (odeke-em) [inline content/node/sequelize/_index.md:120]: As much as we can, please use the tick or X as I did before (I understand this change had just Yes or No) but it is very important to be able to quickly skim over :)

**Review by odeke-em — APPROVED** [review]
Thank you @aabmass for this change, LGTM except for some minor changes!

**Comment by aabmass** [comment]
Checkmarks added

**Comment by aabmass** [comment]
@odeke-em is there a script to update the table of contents?

**Comment by odeke-em** [comment]
Nope, done manually.

**Comment by aabmass** [comment]
Updated TOC

**Comment by odeke-em** [comment]
I’ll just need to Hugo generate and then push to the gh-pages branch of google/sqlcommenter and it’ll all be live.

**Pull request merged**